api/src/main/resources/application.yml

```diff
@@ -1,6 +1,22 @@
+spring:
+  jackson:
+    default-property-inclusion: NON_NULL
+  application:
+    name: eligibility/api
+    boot:
+      admin:
+        url: http://localhost:8100
+
 server:
   port: 8100
 
+logging:
+  pattern:
+    console: "%d{HH:mm:ss.SSS} [%thread] [%X{session.id}][%X{request.id}] %-5level %logger{36} - %msg%n"
+  level:
+    ROOT: ${vcap.services.variable-service.credentials.claimant-root-loglevel:info}
```

Comment:
"credentials.claimant" should be "credentials.eligibility" in this and the line below.

Reply:
Fixed.